src/App.js

```diff
@@ -1,13 +1,23 @@
-import './App.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please do not disable eslint checks. Eslint helps improve your Javascript's quality and keeps it to standard.